test/integration/scripts/.s2i/bin/save-artifacts

```diff
@@ -3,3 +3,7 @@ touch /tmp/src/save-artifacts-invoked
 
 cd /tmp/src
 tar cf - .
+
+# Write some zeros to stdout to mimic the padding that Debian's tar adds
+# https://github.com/openshift/source-to-image/issues/301#issuecomment-263598586
+dd count=1 if=/dev/zero 2> /dev/null
```

Comment:
I first tried `echo junk` as you suggested, but for some reason that was not enough to reproduce the problem...